### ISSUE TITLE
Feature request: Add support for py files

### ISSUE THREAD

**Issue by Chromowolf** [Buizz/EUD-Editor-3#65]
In TriggerEditor of EE3, I could only create/edit/link EPS files. Could you also add support for py files?
There are lots of py files in my project. Currently I couldn't link them to the EE3 project

**Comment by Buizz** [comment]
![image](https://user-images.githubusercontent.com/19189816/132492314-c8c4a27c-ae84-404b-825d-ef9bc953adf6.png)

This button will take you to where the build files are located.
Paste the py code in that location and it will be applied.
py is not supported.

**Comment by Chromowolf** [comment]
> Paste the py code in that location and it will be applied.

Yes, this is exactly what I'm doing as a work-around. But in this case, py files would be mixed with the EE-generated eps files, making it hard to navigate. I would be appreciate if it is possible to add support for py files.
Anyway thanks for your reply

**Comment by Buizz** [comment]
Close for issues cleanup.
If the problem is not solved yet, please post a new issue.
